### PR TITLE
Implement wdpec support for Edge

### PR DESCRIPTION
**Pull request by thejohnjansen**
In order to run the wdspec tests in Edge, we need to implement the WdspecExecutor.

Included in this change is a hack to run the tests even though we do not return a valid w3c session ID yet.

@jgraham - I removed the lines where we Printed the session Id for debugging. You mentioned making the change a little cleaner, but I'm not sure if you wanted something different from what I have here.

---
This change is [Reviewable](https://reviewable.io/reviews/w3c/web-platform-tests/8167)

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build BROKEN

Started: 2017-11-13 18:30:53
Finished: 2017-11-13 18:43:08


<h2>Failing Jobs</h2>
<ul>

<li>tools_unittest in py27</li>

<li>tools_unittest in pypy</li>

</ul>




View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/20528)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/301546671)

File: tools/webdriver/webdriver/client.py [hunk 1]
```diff
@@ -431,6 +431,15 @@ def send_command(self, method, url, body=None):
...
         if "value" in response.body:
             value = response.body["value"]
+            """
+            Edge does not yet return the w3c session ID. 
+            We want the tests to run in Edge anyway to help with REC.
+            In order to run the tests in Edge, we need to hack around
+            bug:
+            https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/14641972
+            """
+            if url == "session" and method == "POST" and "sessionId" in response.body:
```
Comment (jgraham) [inline tools/webdriver/webdriver/client.py]: Can we add an `and "sessionId" not in value` clause here, and log a warning in this case, please?

**Review by jgraham — COMMENTED** [review]
Basically looks great; I'd just like to make sure we don't accidentally end up down the compat branch in too many cases and not notice.

File: tools/webdriver/webdriver/client.py (hunk 1 above)
Comment (thejohnjansen) [reply tools/webdriver/webdriver/client.py]: @jgraham I added the additional clause, but I don't see other logging in this file yet. I don't mind adding it, but I'm not sure about the best way to do it.

Comment (jgraham) [reply tools/webdriver/webdriver/client.py]: `logger.warning("Got legacy new session response")`

The logger is already imported even though it's not used.

Comment (thejohnjansen) [reply tools/webdriver/webdriver/client.py]: I should have mentioned that I tried logger.warning("foo") and logger.debug("foo") but both hit an error:
AttributeError: 'NoneType' object has no attribute 'warning'
Which I think means that the logger was not created successfully.

**Review by jgraham — APPROVED** [review]
OK, let's merge this and I'll try to look at the logger thing.

**Pull request merged**